doc/api/repl.md

```diff
@@ -411,6 +411,8 @@ added: v0.1.91
     `SIGINT` is received, i.e. `Ctrl+C` is pressed. This cannot be used together
     with a custom `eval` function. Defaults to `false`.
 
+If `options` is a string, then it specifies the input prompt.
```

Comment:
Can you elaborate here on what specifying the input prompt means? Perhaps with an example?